pulp_rpm/app/advisory.py

```diff
@@ -167,31 +178,36 @@ def resolve_advisory_conflict(previous_advisory, added_advisory):
                 # prev has more pkgs - exclude new
                 to_exclude.append(added_advisory.pk)
             else:
-                raise AdvisoryConflict(
-                    _(
-                        "Incoming and existing advisories have the same id and timestamp "
-                        "but different and intersecting package lists, "
-                        "and neither package list is a proper subset of the other. "
-                        "At least one of the advisories is wrong. "
-                        f"Advisory id: {previous_advisory.id}"
+                if settings.ALLOW_AUTOMATIC_UNSAFE_ADVISORY_CONFLICT_RESOLUTION:
+                    _do_merge()
+                    raise AdvisoryConflict(
+                        _(
+                            "Incoming and existing advisories have the same id and timestamp "
+                            "but different and intersecting package lists, "
+                            "and neither package list is a proper subset of the other. "
+                            "At least one of the advisories is wrong. "
+                            "Advisory id: {}"
+                        ).format(previous_advisory.id)
```

Comment:
This should be in an `else` condition, shouldn't it?

Reply:
Otherwise LGTM, love the documentation, thank you for all of this work.

Reply:
> This should be in an `else` condition, shouldn't it?

Yes, and it was, once. I rebased this a couple of times between writing it, and pushing it; may have been bitten by one of them. At any rate, good catch!